api/package.json

```diff
@@ -23,8 +23,7 @@
     "express-validator": "^6.10.0",
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.12.2",
-    "puppeteer": "^10.0.0",
-    "puppeteer-cluster": "^0.22.0",
```

Comment:
Curious which PR you intend to submit first. This or https://github.com/Researchify/Researchify/pull/136 ?
The latter relies on `puppeteer` deps.

Reply:
#136 will go in first so there's less conflicts to deal with